tests/lib/managers/webhooks-test.js

```diff
@@ -0,0 +1,158 @@
+/**
+* @fileoverview Webhooks Manager Tests
+* @author ccheng
+*/
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+var sinon = require('sinon'),
+	mockery = require('mockery'),
+	leche = require('leche');
+
+var BoxClient = require('../../../lib/box-client');
+
+
+// ------------------------------------------------------------------------------
+// Helpers
+// ------------------------------------------------------------------------------
+var sandbox = sinon.sandbox.create(),
+	boxClientFake,
+	Webhooks,
+	webhooks,
+	testQS = { testQSKey: 'testQSValue' },
+	testParamsWithQs,
+	WEBHOOKS_ID = '1234',
+	MODULE_WEBHOOKS_PATH = '../../../lib/managers/webhooks';
+
+describe('Webhooks', function() {
+
+	beforeEach(function() {
+		// Setup Environment
+		boxClientFake = leche.fake(BoxClient.prototype);
+		testParamsWithQs = {qs: testQS};
+		// Register Mocks
+		mockery.enable({
+			useCleanCache: true
+		});
+		mockery.registerAllowable('http-status');
+		mockery.registerAllowable('util');
+		mockery.registerAllowable('../util/url-path');
+		mockery.registerAllowable('../util/errors');
+		// Setup File Under Test
+		mockery.registerAllowable(MODULE_WEBHOOKS_PATH);
+		Webhooks = require(MODULE_WEBHOOKS_PATH);
+		webhooks = new Webhooks(boxClientFake);
+	});
+
+	afterEach(function() {
+		sandbox.verifyAndRestore();
+		mockery.deregisterAll();
+		mockery.disable();
+	});
+
+	describe('create()', function() {
+
+		var	ID = '1234',
+			TYPE = 'file',
+			ADDRESS = 'https://www.test.com',
+			TRIGGERS = ['FILE.DOWNLOADED', 'FILE.PREVIEWED'],
+			expectedParams = {
+				body: {
+					target: {
+						id: ID,
+						objectType: TYPE
+					},
+					address: ADDRESS,
+					triggers: TRIGGERS
+				}
+			};
+
+		it('should make POST call to create webhook', function() {
+
+			sandbox.stub(boxClientFake, 'defaultResponseHandler');
+			sandbox.mock(boxClientFake).expects('post').withArgs('/webhooks', expectedParams);
+			webhooks.create(ID, TYPE, ADDRESS, TRIGGERS);
+		});
+
+		it('should call BoxClient defaultResponseHandler method with the callback when response is returned', function(done) {
+			sandbox.mock(boxClientFake).expects('defaultResponseHandler').returns(done);
```

Comment:
We should assert about the argument to defaultResponseHandler in these tests, since that's what the test is about.  Something like `.withArgs(done).returns(done)` would be fine.